Formula/zork.rb

```diff
@@ -0,0 +1,25 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.2.tar.gz"
+  sha256 "169e1848b1e3c503591c23ad4e66ce45e1d5ae617831634e1da9c8fca659e283"
+  head "https://github.com/devshane/zork.git"
+
+  def install
+    system "make", "DATADIR=#{share}", "BINDIR=#{bin}"
+    system "make", "install", "DATADIR=#{share}", "BINDIR=#{bin}", "MANDIR=#{man}"
+  end
+
+  test do
+      test_phrase = <<-EOS.undent
+        Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.
+        You are in an open field west of a big white house with a boarded
+        front door.
+        There is a small mailbox here.
+        >Opening the mailbox reveals:
+          A leaflet.
+      EOS
+      test_phrase << '>'
```

Comment:
The output from `zork` looks like this:
```
"Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.\nYou are in an open field west of a big white house with a boarded\nfront door.\nThere is a small mailbox here.\n>Opening the mailbox reveals:\n  A leaflet.\n>"
```

I might just be missing it but, I could figure out a way to remove the trailing newline with the heredoc format. For the test to pass, the phrase has to end with a `>` without a new line. Include the `>` in the Heredoc I was getting this:
```
❯ brew test  zork
Testing zork
==> Using the sandbox
==> /opt/brew/Cellar/zork/1.0.2/bin/zork
The game is over.
Error: zork: failed
<"Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.\nYou are in an open field west of a big white house with a boarded\nfront door.\nThere is a small mailbox here.\n>Opening the mailbox reveals:\n  A leaflet.\n>\n"> expected but was
<"Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.\nYou are in an open field west of a big white house with a boarded\nfront door.\nThere is a small mailbox here.\n>Opening the mailbox reveals:\n  A leaflet.\n>">.
```

Is there a cleaner way to do this?